README_PODMAN.md

```diff
@@ -0,0 +1,44 @@
+# Using the dnsname plugin with Podman
+
+The purpose of the *dnsname* plugin is to allow containers to resolve each other by name.  The plugin adds each
```

Comment:
The *dnsname* plufin allows containers to resolve each other by name. The plugin adds ...